Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -800,8 +794,17 @@
 #define ENABLE_APP_PRIVACY_REPORT 1
 #endif
 
-#if !defined(ENABLE_APP_HIGHLIGHTS) \
-    && (PLATFORM(MAC) || PLATFORM(IOS) || PLATFORM(VISION))
+#if PLATFORM(VISION)
+#define ENABLE_APP_HIGHLIGHTS 0
```

Comment:
Now that we're upstreaming these, we should make them conditions on the #defines that enable the feature, not have this fun/unusual mass-override, right?